src/globals.ts

```diff
@@ -9,11 +9,11 @@ export const ERR_NO_NFTS = new Error('No NFTs Found:( Are you on the right netwo
 // ---------------------------------------  defaults
 
 export const DEFAULTS = {
-  CLUSTER: 'mainnet',
+  CLUSTER: 'devnet',
 
   OWNER: 'AEahaRpDFzg74t7NtWoruabo2fPJQjKFM9kQJNjH7obK',
-  CREATOR: 'BDYYJ1VzPDXwJoARMZNnN4MX4cZNjVvc5DfFaKzgrruz',
-  AUTHORITY: 'Edy3xedybSQYAE6eboza5YEcoJan91nfSd3jUppHz7u1',
+  CREATOR: '2WJwPd4EG7wZmx2BjAASoMh1pfRYLLb1CHj4oMooPKb5',
```

Comment:
lets change this to the community wallet : 9px36ZsECEdSbNAobezC77Wr9BfACenRN1W8X7AUuWAb